src/crashhandler_unix.cpp

```diff
@@ -230,7 +230,7 @@ namespace g3 {
          std::cerr << "\n\n" << __FUNCTION__ << ":" << __LINE__ << ". Exiting due to " << level.text << ", " << signal_number << "   \n\n" << std::flush;
 
 
-         kill(getpid(), signal_number);
+         raise(signal_number);
```

Comment:
Thank you for the detailed explanation. I'm currently traveling and will review this once once I'm back next week